incubator/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/config/SchemaRegistryOptionsConfigAdapter.java

```diff
@@ -78,6 +87,15 @@ public OptionsConfig adaptFromJson(
             {
                 options.context(object.getString(CONTEXT));
             }
+
+            if (object.containsKey(MAX_AGE_NAME))
+            {
+                options.maxAge(Duration.ofSeconds(object.getJsonNumber(MAX_AGE_NAME).longValue()));
+            }
+            else
+            {
+                options.maxAge();
+            }
```

Comment:
This else clause should be removed, then defaulting happens in the builder build method as needed.